test/ibmq/test_account_client_v2.py

```diff
@@ -395,3 +395,19 @@ def test_api_version(self, qe_token, qe_url):
         api = AuthClient(qe_token, qe_url)
         version = api.api_version()
         self.assertIsNotNone(version)
+
+    @requires_qe_access
+    @requires_new_api_auth
+    def test_user_urls(self, qe_token, qe_url):
+        """Check the user urls of the QX API."""
+        api = AuthClient(qe_token, qe_url)
+        user_urls = api.user_urls()
+        self.assertIsNotNone(user_urls)
```

Comment:
Given that the docstring says the returned dictionary supports `http` and `ws` as keys, it'd be nice to verify they exit